src/include/execution/vm/bytecodes.h

```diff
@@ -172,6 +172,15 @@ namespace terrier::execution::vm {
                                                                                                                       \
   /* SQL type comparisons */                                                                                          \
   F(ForceBoolTruth, OperandType::Local, OperandType::Local)                                                           \
+  F(IsNull, OperandType::Local, OperandType::Local)                                                                   \
+  F(IsNotNull, OperandType::Local, OperandType::Local)                                                                \
```

Comment:
You can remove `IsNull` and `IsNotNull` and use `ValIsNull` and `ValIsNotNull` - they're meant to serve the same purpose. Tweak the latter two bytecodes to use primitive booleans.